SIT.Manager/Views/Installation/ConfigureSitView.axaml

```diff
@@ -116,8 +116,24 @@
 						</StackPanel>
 				</Border>
 			</StackPanel>
+			<StackPanel>
+				<TextBlock Text="{DynamicResource ConfigureViewPersonalizationOptions}" Classes="FrameHeading"/>
+				<Border Classes="StandardFrame">
+		<CheckBox HorizontalAlignment="Left"
+				  Margin="0,0,0,0"
+				  ToolTip.Tip="{DynamicResource ConfigureSitViewEFTSettingsCheckboxTip}"
+			      IsChecked="{Binding CurrentInstallProcessState.CopyEftSettings}">
+			<CheckBox.Content>
+				<Grid ColumnDefinitions="Auto,*">
+					<ui:SymbolIcon Grid.Column="0" Symbol="Settings" Margin="0,0,0,0"/>
+					<TextBlock Grid.Column="1" Text="{DynamicResource ConfigureSitViewEFTSettingsCheckboxTitle}" Margin="5,0,0,0" VerticalAlignment="Center" TextTrimming="CharacterEllipsis"/>
+				</Grid>
+			</CheckBox.Content>
+		</CheckBox>
+				</Border>
+			</StackPanel>
 		</StackPanel>
```

Comment:
Just a slight niggle about this. Could this stack panels contents be moved inside the stackpanel above?
Also a few pixels of margin between the borders would be nice :3